docs/specifications/tests/Consistency-TP/consistency04.md

```diff
@@ -21,16 +21,21 @@ consistent between all the authoritative name servers.
 
 ### Ordered description of steps to be taken to execute the test case
 
-1. Obtain the list of name servers from [Method2](../Methods.md) and
-   [Method3](../Methods.md).
-2. Retrieve the NS RR set from all the name servers. 
-3. If the NS RR set is not give the same answer from all the name
+1. Obtain the list of name servers for the domain name to be tested 
+   from [Method2](../Methods.md) (delegation) and
+   [Method3](../Methods.md) (in child zone).
+2. Retrieve the IP addresses for the obtained name servers using
```

Comment:
I suggest a change to the wording:
For each in-zone name server, retrieve its IP addresses using Method4 and Method5.